client/card-readers/preview-receipt/printed-receipt-previewer.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import React from 'react';
+import { createPortal } from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+interface PrintedReceiptPreviewerProps {
+	receiptHtml: string;
+}
+
+interface IFrameComponentProps {
+	title: string;
+	children: React.ReactNode;
+}
+
+const IFrameComponent = ( { title, children }: IFrameComponentProps ) => {
```

Comment:
I suggest exporting `IFrameComponent` as a separate component in the `components` folder for reusability.

Reply:
Thanks @mordeth I see your point. I was hesitating to do that because I don't think this is a generic component that could fit there. The way I see it, the iframe I am using here is tied to the preview functionality. It does not care about the regular use case of an iframe...It does not care about `src`, has a `portal` and `ref`...I hope you see my point. Maybe I could give it a clever name to differentiate it from a regular iframe component, but I have my doubts about it is going to be ever reused?

Please let me know your thoughts